microsoft-edge/toc.yml

```diff
@@ -373,22 +373,44 @@
 # equivalent card: Progressive Web Apps =======================================
     - name: Progressive Web Apps
       items:
-        - name: Progressive Web Apps overview
+        - name: Overview of Progressive Web Apps (PWAs)
           href: progressive-web-apps-chromium/index.md
+        - name: The user experience of PWAs
+          href: progressive-web-apps-chromium/ux.md
+        - name: How to
+          items:
+            - name: Get started with PWAs
+              href: progressive-web-apps-chromium/how-to/index.md
+            - name: Web app manifest
+              href: progressive-web-apps-chromium/how-to/webappmanifests.md
+            - name: Service Worker
```

Comment:
```suggestion
            - name: Service Workers for network requests and push notifications
```

Reply:
@captainbrosset - or, change to "Service workers".  The title of the article is
Use Service Workers to manage network requests and push notifications
https://review.docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/serviceworker?branch=pr-en-us-1429